cluster/images/etcd/rollback/rollback.go

```diff
@@ -26,7 +26,10 @@ import (
 	"strings"
 	"time"
 
-	wal237 "k8s.io/kubernetes/third_party/forked/etcd237/wal"
+	// Uncomment when you want to rollback to 2.2.1 version.
+	oldwal "k8s.io/kubernetes/third_party/forked/etcd221/wal"
+	// Uncomment when you want to rollback to 2.3.7 version.
+	// oldwal "k8s.io/kubernetes/third_party/forked/etcd237/wal"
```

Comment:
We can import two packages. Adding a switch statement in the code to switch which pkg of wal to call based on user input.